src/app/components/fee-row/fee-row.tsx

```diff
@@ -43,6 +42,10 @@ export function FeeRow(props: FeeRowProps): JSX.Element {
     if (!input.value && !isCustom && feeEstimations.length) {
       helpers.setValue(microStxToStx(feeEstimations[1].fee).toNumber());
     }
+    // Set fee to 0 if the tx is sponsored
```

Comment:
The code reads `if sponsored, set value 0`, imo this comment adds little.

If it isn't clear what is being set to 0 (the fee), rename helpers to `feeHelpers`

Reply:
Ok, thx, I'll remove it. 👍